packages/xchain-thorchain/__tests__/client.test.ts

```diff
@@ -1,4 +1,4 @@
-import { proto } from '@cosmos-client/core'
+import { proto } from '@cosmos-client/core/cjs/module'
```

Comment:
Q: Why not `import cosmosclient from '@cosmos-client/core'` (w/o cjs/module) as we have it in other sources?